packages/react-dom-bindings/src/client/validateDOMNesting.js

```diff
@@ -483,28 +553,50 @@ function validateDOMNesting(
       // stack has more useful context for nesting.
       // TODO: Format this as a linkified "diff view" with props instead of
       // a stack trace since the stack trace format is now for owner stacks.
-      console['error'](
+      console.error(
         'In HTML, %s cannot be a child of <%s>.%s\n' +
           'This will cause a hydration error.%s',
         tagDisplayName,
         ancestorTag,
         info,
-        getCurrentParentStackInDev(),
+        ancestorDescription,
       );
     } else {
       // Don't transform into consoleWithStackDev here because we add a manual stack.
       // We use the parent stack here instead of the owner stack because the parent
       // stack has more useful context for nesting.
       // TODO: Format this as a linkified "diff view" with props instead of
       // a stack trace since the stack trace format is now for owner stacks.
-      console['error'](
```

Comment:
You left the comment which says not to transform the console method but I think it’s no longer valid